robocop/checkers/spacing.py

```diff
@@ -0,0 +1,131 @@
+"""
+Spacing checkers
+"""
+from robot.parsing.model.blocks import TestCase, Keyword
+from robot.parsing.model.statements import EmptyLine, Comment
+from robocop.checkers import RawFileChecker, VisitorChecker
+from robocop.rules import RuleSeverity
+
+
+def register(linter):
+    linter.register_checker(InvalidSpacingChecker(linter))
+    linter.register_checker(MissingTrailingBlankLineChecker(linter))
+
+
+class InvalidSpacingChecker(RawFileChecker):
+    """ Checker for invalid spacing. """
+    rules = {
+        "1001": (
+            "trailing-whitespace",
+            "Trailing whitespace at the end of line",
+            RuleSeverity.WARNING
+        ),
+        "1002": (
+            "missing-trailing-blank-line",
+            "Missing trailing blank line at the end of file",
+            RuleSeverity.WARNING
+        )
+    }
+
+    def __init__(self, *args):
+        self.lines = []
+        super().__init__(*args)
+
+    def parse_file(self):
+        self.lines = []
+        super().parse_file()
+        if self.lines and not self.lines[-1].endswith('\n'):
+            self.report("missing-trailing-blank-line", lineno=len(self.lines), col=0)
+
+    def check_line(self, line, lineno):
+        self.lines.append(line)
+
+        stripped_line = line.rstrip('\n')
+        if stripped_line:
+            if stripped_line[-1] == ' ':
```

Comment:
```suggestion
        if stripped_line and stripped_line[-1] == ' ':
```

Reply:
Yeah, we talked about it and I forgot to change it. Thanks 👍